static/app/utils/replays/replayReader.tsx

```diff
@@ -282,15 +370,15 @@ export default class ReplayReader {
   );
 
   getChapterFrames = memoize(() =>
-    [
+    this._trimFramesToClipWindow([
       ...this.getPerfFrames(),
       ...this._sortedBreadcrumbFrames.filter(frame =>
         ['replay.init', 'replay.mutations', 'replay.hydrate-error'].includes(
           frame.category
         )
       ),
       ...this._errors,
-    ].sort(sortFrames)
+    ]).sort(sortFrames)
```

Comment:
It would be better to call `this._trimFramesToClipWindow(this._errors)` up inside of `_applyClipWindow`. That way all the lists are trimmed before we get down here into the various `get*` filter methods.

Reply:
Trimmed the errors in `_applyClipWindow`. Still need to trim in this public function though because we do need the historical breadcrumb frames for knowing the current URL.